package.json

```diff
@@ -98,7 +98,7 @@
       "examples/passport/**",
       "examples/orderbook/**",
       "examples/checkout/**",
-      "examples/blockchain-data/**",
```

Comment:
@zaidarain1 I had to update this in order for the `/examples/blockchain-data` package to be picked up as an included workspace. Do you see any issues with this?